public/js/main.js

```diff
@@ -123,11 +123,12 @@ function processKeyPoints(combinedKeyPoints) {
   }
 
   if (state.isInferenceOn) {
-    if (facePoints !== undefined && handPoints !== undefined) {
+    // TODO Look into merging/removing this condition to the line above to avoid unnecessary computations
+    if (facePoints !== undefined && handPoints !== null) {
       computeInference(facePoints, handPoints).then((inference) =>
         updateInferenceText(inference[0])
       );
-    }
+    } else document.getElementById('inference-txt').innerHTML = '0.00 %';
```

Comment:
just the note that this is gonna get overridden by https://github.com/misterpeddy/hands-down/issues/41

Reply:
That's fine, this bit was just to have it set to 0 when the hands were off-screen.